tests/integration/scenarios/in_forward/tests/test_in_forward_001.py

```
@@ -408,6 +421,181 @@ def _zstd_bytes(data):
+
+    if data_limit_kb is not None:
+        def set_data_limit():
+            import resource
+
+            _, hard_limit = resource.getrlimit(resource.RLIMIT_DATA)
+            resource.setrlimit(resource.RLIMIT_DATA, (data_limit_kb * 1024, hard_limit))
+
+        preexec_fn = set_data_limit
+
+    output = open(log_file, "a", encoding="utf-8")
+    process = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)